code/defines/obj/weapon.dm

```diff
@@ -12,6 +12,49 @@
 	w_class = W_CLASS_SMALL
 	attack_verb = list("calls", "rings", "dials")
 	hitsound = 'sound/weapons/ring.ogg'
+	var/obj/landline/landline
+	
+/obj/item/weapon/phone/New()
+	landline = new /obj/landline/red (src,src)
+	var/list/a = list("alpha","bravo","charlie","delta","echo","foxtrot","golf","hotel","india")
+	var/list/b = list("anton","boris","vasilij","grigorij","dimitrij","elena","zhenja","ivan","nikolaj")
+	var/list/c = list("1","2","3","4","5","6","7","8","9")
+	name += " [pick(a)]-[pick(b)]-[pick(c)]"
```

Comment:
although unlikely, there is nothing to prevent two phones from picking the exact same designation, which can happen with roughly a 4% chance based on the number of RC consoles on a map

you have some options on fixing this like checking against existing landlines or pick_n_take

Reply:
This applies to red phones only. Generally, only 1-2 red phones (centcom and HM easter egg vault) exist per round, so is that really an issue?..

Reply:
Ah, I thought this was being assigned per RC. Odds are quite low with just 2 but why do a lazy solution when you could do it right and NEVER have a problem?

Reply:
True, I respect your mindset.

Reply:
how much global variable bloat can i get away with, anyway?
and while i'm at it, should i move the RC lists from message.dm into global_lists.dm?

Reply:
currently we have no policy limiting adding global vars (funny enough this has been a minor discussion lately but don't worry about it for now)

you may put the lists into global_lists if you deem it valuable, but plenty of global lists are not - your choice